pkg/cli/start.go

```diff
@@ -386,7 +387,7 @@ func runStart(cmd *cobra.Command, args []string) error {
 			var buf bytes.Buffer
 			info := build.GetInfo()
 			tw := tabwriter.NewWriter(&buf, 2, 1, 2, ' ', 0)
-			fmt.Fprintf(tw, "CockroachDB node starting at %s\n", timeutil.Now())
+			fmt.Fprintf(tw, "CockroachDB node starting at %s (took %s)\n", timeutil.Now(), timeutil.Since(tBegin))
```

Comment:
I have a small nit against the over-precision of `time.Time.String()`. Better in my opinion to display the time as `fmt.Sprintf("%0.1fs", timeutil.Since(tBegin).Seconds())`.